examples/migraphx/README.md

```diff
@@ -0,0 +1,7 @@
+# AMD MIGraphX usage and utilities
+
+- [C++ Parse, Load, and Save Graph Programs](./cpp_parse_load_save)
```

Comment:
For the readme file, can we mention "cmake .. -DCMAKE_PREFIX_PATH="migraphx_install_loc"?

Reply:
I think this is covered in main readme file of migraphx, and each app in /migraphx/ examples has their own explanation.